### PR TITLE
tls: refresh tls certs for test

### PR DESCRIPTION
**Pull request by wbpcode**
Commit Message: tls: refresh tls certs for test
Additional Description: 

`ca_cert.pem` was expired. This PR refreshs all files by `certs.sh`.

Risk Level: Low. For test only.
Testing: n/a.
Docs Changes: n/a.
Release Notes: n/a.
Platform Specific Features: n/a.

### CODE REVIEW

**Comment by wbpcode** [comment]
```
$ openssl x509 -in ./test/extensions/transport_sockets/tls/test_data/ca_cert.pem -noout -dates
notBefore=Aug 20 16:57:46 2020 GMT
notAfter=Aug 20 16:57:46 2022 GMT
```

File: test/extensions/transport_sockets/tls/test_data/i1_cert.pem [hunk 1]
```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```
Comment (adisuissa) [inline test/extensions/transport_sockets/tls/test_data/i1_cert.pem:1]: Do you happen to know why this file (and a few others that follow) was empty before?

**Review by phlax — APPROVED** [review]
lgtm, thanks @wbpcode

**Review by adisuissa — COMMENTED** [review]
Thanks! Just encountered this myself :)

File: test/extensions/transport_sockets/tls/test_data/i1_cert.pem (hunk 1 above)
Comment (wbpcode) [reply test/extensions/transport_sockets/tls/test_data/i1_cert.pem:1]:
```
generate_cert_chain() {
    local certname
    local ca_name="${1}"
    rm test_long_cert_chain.pem
    touch test_long_cert_chain.pem
    for x in {1..4}; do
        certname="i$x"
        if [[ $x -gt 1 ]]
        then
            ca_name="i$((x - 1))"
        fi
        echo "$x: $certname $ca_name"
        generate_ca $certname $ca_name
    done
    for x in {1..3}; do
        cat "i${x}_cert.pem" >> test_long_cert_chain.pem
    done
    mv i4_cert.pem test_random_cert.pem
}
```
They seems like the intermediate files that used to generate  `test_long_cert_chain.pem` and `test_random_cert.pem`.

Seems that the `certs.sh` didn't clean these files.

Comment (wbpcode) [reply test/extensions/transport_sockets/tls/test_data/i1_cert.pem:1]: Should I clean the files directly or clean these files and update `certs.sh` in new separated PR?

Comment (wbpcode) [reply test/extensions/transport_sockets/tls/test_data/i1_cert.pem]: All these files are deleted and I will update the `certs.sh` with new PR.

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Comment by ggreenway** [comment]
Question: why do we check in the generated certs, instead of generating them as part of the build?

**Comment by wbpcode** [comment]
> Question: why do we check in the generated certs, instead of generating them as part of the build?

I have no idea why we use the generated certs. May be to avoid complex gen rules in runtime?

**Pull request merged**